src/app/public/components/about/about.component.html

```diff
@@ -157,27 +157,30 @@ <h3>Klima Campus</h3>
 			<h1 class="h1-black" style="margin-top: 1em">Du willst mitmachen?</h1>
 
 			<div>
-				<p class="text-big">Klicke auf die Kacheln, um weitere Informationen und Unterstützung zu bekommen.</p>
+				<p class="text-big">
+					Klicke auf die Kacheln, um in Kontakt mit uns zu kommen.
+				</p>
 
 				<div class="join-us-group-cards">
-					<div class="join-us-card" style="width: 60%; background-color: var(--color-green)">
+					<a class="join-us-card tw-w-[60%] tw-bg-green" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
 						<h2 class="h2-bold">Open Educational Badges jetzt nutzen</h2>
-					</div>
-					<div class="join-us-card" style="width: 40%; background-color: var(--color-pink)">
-						<h2 class="h2-bold" style="color: #ffffff">Ein Projekt rund um Open Badges durchführen</h2>
-					</div>
+					</a>
+
+					<a class="join-us-card tw-w-[40%] tw-bg-pink" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
+						<h2 class="h2-bold">Ein Projekt rund um OEB durchführen</h2>
+					</a>
 				</div>
 
 				<div class="join-us-group-cards">
-					<div class="join-us-card" style="width: 30%; background-color: var(--color-purple)">
-						<h2 class="h2-bold" style="color: #ffffff">Open Badges fördern</h2>
-					</div>
-					<div class="join-us-card" style="width: 40%; background-color: var(--color-yellow)">
-						<h2 class="h2-bold" style="color: #ffffff">Die Plattform oder ein Feature fördern</h2>
-					</div>
-					<div class="join-us-card" style="width: 30%; background-color: var(--color-red)">
-						<h2 class="h2-bold" style="color: #ffffff">Erweiterung des Plugins fördern</h2>
-					</div>
+					<a class="join-us-card tw-w-[30%] tw-bg-purple" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
+						<h2 class="h2-bold !tw-text-white">OEB fördern</h2>
+					</a>
+					<a class="join-us-card tw-w-[40%] tw-bg-yellow" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
+						<h2 class="h2-bold">Aktiv mitmachen bei Open Educational Badges</h2>
+					</a>
+					<a class="join-us-card tw-w-[30%] tw-bg-red" href='mailto:{{mailAdrees}}?subject={{mailBody}}'>
+						<h2 class="h2-bold !tw-text-white">Ein individuelles PlugIn anfragen</h2>
```

Comment:
I'd typically write it *Plugin* and not *PlugIn*

Reply:
We can suggest this to Mycelia team ;)

Reply:
I've already contacted them to choose between `plug-In` or `plugin`

Reply:
The change was resolved.